lib/jekyll-import/importers/blogger.rb

```diff
@@ -42,12 +44,12 @@ def self.process(options)
         listener = BloggerAtomStreamListener.new
 
         listener.leave_blogger_info = !options.fetch("no-blogger-info", false),
-        listener.comments = options.fetch("comments", false),
+                                      listener.comments = options.fetch("comments", false),
```

Comment:
why are there commas at the end of the line here and the line above?

Reply:
I think the comma on line 44/46 is probably wrong? These should each be separate lines I think without commas since they're declarations